sdk/ai/azure-ai-generative/setup.py

```diff
@@ -88,7 +88,7 @@
             "GitPython>=3.1,<4"
         ],
         "evaluate": [
-            "azureml-metrics[prompt-flow]==0.1.0.105202174",
+            # "azureml-metrics[prompt-flow]==0.1.0.105202174",
```

Comment:
Can you update this to match what's in the cli sdk v2 repo?

Reply:
done